ports/cppwinrt/vcpkg.json

```
@@ -1,7 +1,15 @@
 {
   "name": "cppwinrt",
-  "version-string": "windows-sdk",
+  "version": "2.0.220418.1",
   "description": "C++/WinRT is a standard C++ language projection for the Windows Runtime.",
-  "homepage": "https://github.com/Microsoft/cppwinrt",
-  "supports": "windows"
+  "homepage": "https://github.com/microsoft/cppwinrt",
+  "documentation": "https://docs.microsoft.com/windows/uwp/cpp-and-winrt-apis/",
+  "license": "MIT",
+  "supports": "windows",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
```

```suggestion
  "supports": "windows"
```
This port doesn't use `vcpkg_cmake_config_fixup()`, so it doesn't dependency `vcpkg-cmake-config`.